### PR TITLE
Add docs to env

### PR DESCRIPTION
**Pull request by odinuge**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,21 +42,38 @@ $ yarn ssr # or yarn ssr:staging
...
-- SSR_API_URL
-- SSR_BASE_URL
+- `NODE_ENV` _(optional)_
+  - `development` during dev and `production` when
+- `API_URL` _(optional)_
+  - Url to the LEGO api. Usally ends with `/api/v1`
+- `WS_URL` _(optional)_
+  - Url to the LEGO websocket endpoint
+- `BASE_URL` _(optional)_
+  - Url to the base of the LEGO api. Usally just the root domain.
```
Comment (SmithPeder) [inline README.md]: usually*

File: README.md [hunk 2]
```diff
@@ -42,21 +42,38 @@ $ yarn ssr # or yarn ssr:staging
...
-- ENVIRONMENT
-- HOST
-- PORT
-- SERVER_RAVEN_DSN
-- SSR_API_URL
-- SSR_BASE_URL
+- `NODE_ENV` _(optional)_
+  - `development` during dev and `production` when
+- `API_URL` _(optional)_
+  - Url to the LEGO api. Usally ends with `/api/v1`
```
Comment (SmithPeder) [inline README.md]: usually*

File: README.md [hunk 3]
```diff
@@ -42,21 +42,38 @@ $ yarn ssr # or yarn ssr:staging
...
-- RAVEN_DSN
-- RELEASE
-- ENVIRONMENT
-- HOST
-- PORT
-- SERVER_RAVEN_DSN
-- SSR_API_URL
-- SSR_BASE_URL
+- `NODE_ENV` _(optional)_
+  - `development` during dev and `production` when
```
Comment (SmithPeder) [inline README.md]: when what? Seems like an unfinished sentence!

File: README.md [hunk 4]
```diff
@@ -42,21 +42,38 @@ $ yarn ssr # or yarn ssr:staging
...
+- `CAPTCHA_KEY` _(optional)_
+  - More info here: <https://developers.google.com/recaptcha/docs/display>
+- `STRIPE_KEY` _(optional)_
+  - More info here: <https://stripe.com/docs/keys>
+- `RAVEN_DSN` _(optional)_
+  - More info here: https://github.com/getsentry/sentry-javascript
+- `SERVER_RAVEN_DSN`_(optional)_
+  - More info here: https://github.com/getsentry/raven-node
+- `RELEASE`_(optional)_
+  - Relase version used when sending exceptions to Sentry. Injected when building docker images
```
Comment (SmithPeder) [inline README.md]: Release*

Comment (odinuge) [reply README.md]: :+1:

File: README.md (hunk 3 above)
Comment (odinuge) [reply README.md]: :skra:

File: README.md (hunk 2 above)
Comment (odinuge) [reply README.md]: :+1:

File: README.md (hunk 1 above)
Comment (odinuge) [reply README.md]: :+1:

File: README.md (hunk 3 above)
Comment (odinuge) [reply README.md]: 10/10 bad sentence

**Pull request merged**